### ISSUE TITLE
Flatpak not working with Wayland

### ISSUE THREAD

**Issue by Ezwen** [ajbura/cinny#582]
### Describe the bug

When running Cinny as a Flatpak on a desktop running under Wayland (instead of X11), Cinny instantly crashes, with the following line of system log:

```
segfault at e0 ip 00007f5934a9b144 sp 00007f593094e6b8 error 4 in libX11.so.6.4.0[7f5934a82000+94000]
```

### Reproduction

1. Run a Fedora 36 GNOME session with Wayland (which is the case per default with the default Fedora 36 flavor)
2. Install Cinny as a Flatpak
3. Try to start Cinny

At this point, Cinny does not start, and running `journalctl -b | grep -i cinny` I can find:
```
segfault at e0 ip 00007f5934a9b144 sp 00007f593094e6b8 error 4 in libX11.so.6.4.0[7f5934a82000+94000]
```

### Expected behavior

Cinny should start without crashing.

### Platform and versions

```shell
1. OS: Fedora 36, GNOME, Wayland
2. Browser: N/A (using the desktop version as a Flatpal)
3. Cinny: v2.0.3
4. Homeserver: private
```

### Additional context

This problem does not appear with running GNOME with X11.

**Comment by jakobkmar** [comment]
Workaround is to disable the `fallback-x11` socket and enable the `x11` socket.
If you want to do it using a graphical client, this can be done using Flatseal:
![image](https://user-images.githubusercontent.com/52456572/170562944-0257a702-0d50-4e8f-86f2-6cc8eb280b06.png)

However it would be the best if Cinny supports Wayland - and does not have to run through XWayland.

Until that does not happen, the sockets in [the Manifest](https://github.com/flathub/in.cinny.Cinny/blob/master/in.cinny.Cinny.yml) should probably be updated and pass x11 directly (instead of fallback-x11). Pinging @kfiven here :)

**Comment by kfiven** [comment]
Done. I have removed the wayland socket and revert back to x11.

**Comment by reesericci** [comment]
Without Wayland support, Cinny is a blurry mess on my desktop because my screen requires fractional scaling (175%), something that is only supported by Wayland. Is there any update on a timeline for proper Wayland support?

**Comment by kfiven** [comment]
The reason Flatpak doesn't work well is because there are issues with Tauri<=>Wayland. And we have no control over that.

**Comment by reesericci** [comment]
That's a real shame to hear.

**Comment by kfiven** [comment]
should be fixed in 2.2.0

**Comment by jhebert8** [comment]
It still isn't fixed.
